src/ViewArticle/ViewArticleStyles/ViewArticle.scss

```diff
@@ -0,0 +1,95 @@
+@import '../../commons/assets/variables/font.scss';
+@import '../../commons/assets/variables/colors.scss';
+
+.article_title{
+    color: #00a4d3;
+    font-family:Lato;
+    font-weight: light;
```

Comment:
@jennizalwango kindly consider calling this color and font above from the variables you imported in line two of this file. You also will no longer need this `font-weight` property then.

Reply:
Actually l  have worked on this as well although l will need the `font-weight` since the variable am import is not configured to have `font-weight` it only takes in the `font family`